### ISSUE TITLE
dashboard 路由数据编辑不支持chaitin-waf插件,报:schema validate failed: schema not found, path: plugins.chaitin-waf

### ISSUE THREAD

**Issue by lan11** [apache/apisix-dashboard#2903]
### Issue description

启用chaitin-waf插件后对路由进行数据编辑配置chaitin-waf配置信息会报:schema validate failed: schema not found, path: plugins.chaitin-waf

### Expected behavior

支持配置chaitin-waf插件

### How to Reproduce

数据编辑器配置waf插件:  
"plugins": {
    "chaitin-waf": {
      "append_waf_debug_header": false,
      "append_waf_resp_header": true
    }
  },

### Screenshots

![image](https://github.com/apache/apisix-dashboard/assets/41682375/d13e1559-fda3-4ab0-983a-ba26d11e49c0)

### Environment

docker部署
apisix-dashboard镜像版本:3.0.1-alpine
apisix镜像版本:3.7.0-debian

**Comment by lan11** [comment]
在项目/usr/local/apisix-dashboard/conf/schema.json中plugins节点加入chaitin-waf配置信息,可以解决该问题,添加内容如下:
"chaitin-waf": {
			"priority": 2700,
			"metadata_schema": {
				"properties": {
					"nodes": {
						"items": {
							"properties": {
								"host": {
									"pattern": "^\\*?[0-9a-zA-Z-._\\[\\]:]+$",
									"type": "string"
								},
								"port": {
									"minimum": 1,
									"default": 80,
									"type": "integer"
								}
							},
							"required": ["host"],
							"type": "object"
						},
						"minItems": 1,
						"type": "array"
					},
					"config": {
						"properties": {
							"req_body_size": {
								"default": 1024,
								"type": "integer"
							},
							"read_timeout": {
								"default": 1000,
								"type": "integer"
							},
							"send_timeout": {
								"default": 1000,
								"type": "integer"
							},
							"keepalive_size": {
								"default": 256,
								"type": "integer"
							},
							"keepalive_timeout": {
								"default": 60000,
								"type": "integer"
							},
							"connect_timeout": {
								"default": 1000,
								"type": "integer"
							}
						},
						"default": {},
						"type": "object"
					}
				},
				"required": ["nodes"],
				"type": "object"
			},
			"version": 0.1,
			"schema": {
				"properties": {
					"_meta": {
						"properties": {
							"filter": {
								"type": "array",
								"description": "filter determines whether the plugin needs to be executed at runtime"
							},
							"disable": {
								"type": "boolean"
							},
							"priority": {
								"type": "integer",
								"description": "priority of plugins by customized order"
							},
							"error_response": {
								"oneOf": [{
									"type": "string"
								}, {
									"type": "object"
								}]
							}
						},
						"type": "object"
					},
					"match": {
						"items": {
							"properties": {
								"vars": {
									"type": "array"
								}
							},
							"type": "object"
						},
						"type": "array"
					},
					"append_waf_debug_header": {
						"default": false,
						"type": "boolean"
					},
					"append_waf_resp_header": {
						"default": true,
						"type": "boolean"
					},
					"config": {
						"properties": {
							"req_body_size": {
								"type": "integer"
							},
							"read_timeout": {
								"type": "integer"
							},
							"send_timeout": {
								"type": "integer"
							},
							"keepalive_size": {
								"type": "integer"
							},
							"keepalive_timeout": {
								"type": "integer"
							},
							"connect_timeout": {
								"type": "integer"
							}
						},
						"type": "object"
					}
				},
				"$comment": "this is a mark for our injected plugin schema",
				"type": "object"
			}
		},

**Comment by hanqingwu** [comment]
Yes,   conf/schema.json have not include  more plugins schema ,  So  there is  only appropriate way to  add these plugins by apisix  admin api .

**Comment by OnlyPiglet** [comment]
be careful need to add  ``- chaitin-waf`` at  ``/usr/local/apisix/dashboard/conf/config.yaml``  or the plugin don't show on page , code is here 


<img width="711" alt="image" src="https://github.com/user-attachments/assets/b7a6cab8-36f9-4401-9316-445678176bb6" />

**Comment by juzhiyuan** [comment]
Hi, we will release a new dashboard to address legacy issues. Refer to https://github.com/apache/apisix-dashboard/issues/2981
